app/test_main.py

```diff
@@ -1,3 +1,18 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, human_age", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (40, 40, [6, 5]),
+    (100, 100, [21, 17]),
+])
```

Comment:
The test cases provided cover a range of typical inputs, but it's important to include edge cases such as negative numbers or very large numbers to ensure the function handles unexpected inputs gracefully. According to the checklist, you should also verify that the function raises the correct exception when receiving incorrect data types.